packages/shared/src/components/Sidebar.spec.tsx

```diff
@@ -1,23 +1,46 @@
 import React from 'react';
 import nock from 'nock';
 import { render, RenderResult, screen, waitFor } from '@testing-library/react';
+import { QueryClient, QueryClientProvider } from 'react-query';
 import AuthContext from '../contexts/AuthContext';
 import defaultUser from '../../__tests__/fixture/loggedUser';
 import { LoggedUser } from '../lib/user';
 import Sidebar from './Sidebar';
 import SettingsContext, {
   SettingsContextData,
 } from '../contexts/SettingsContext';
+import { mockGraphQL } from '../../__tests__/helpers/graphql';
+import { FEED_SETTINGS_QUERY } from '../graphql/feedSettings';
+import { getFeedSettingsQueryKey } from '../hooks/useMutateFilters';
+import AlertContext, { AlertContextData } from '../contexts/AlertContext';
+import OnboardingContext from '../contexts/OnboardingContext';
+import { waitForNock } from '../../__tests__/helpers/utilities';
 
+let client: QueryClient;
+const updateAlerts = jest.fn();
 const toggleOpenSidebar = jest.fn();
+const incrementOnboardingStep = jest.fn();
```

Comment:
Be aware we plan to remove the onboarding all together.
See slack discussion on Team Engineering

Reply:
Thank you! I will make sure to avoid these cases.